Dockerfile

```diff
@@ -93,7 +93,7 @@ RUN set -eux; \
 	;
 
 # prevent the reinstallation of vendors at every changes in the source code
-COPY package.json yarn.* ./
+COPY package.json yar[n].lock ./
```

Comment:
This is a wildcard that accepts "n" char and only "n" char XD I know it's a bit hacky but this workaround allows us to copy `yarn.lock` if exists or skip it if not. Without any wildcard (previously wildcard was an asterisk), building the docker image fails because can't find the `yarn.lock` file.

Reply:
Indeed, it's tricky, but now I understand, thanks :D